app/presenters/news_article_structured.rb

```diff
@@ -0,0 +1,62 @@
+class NewsArticleStructured
+  def initialize(presenter)
+    @presenter = presenter
+  end
+
+  def structured_data
+    return {} unless enough_structured_data?
+
+    # http://schema.org/NewsArticle
+    {
+      "@context" => "http://schema.org",
+      "@type" => "NewsArticle",
+      "mainEntityOfPage" => {
+        "@type" => "WebPage",
+        "@id" => page_url,
+      },
+      "headline" => presenter.title,
+      "datePublished" => presenter.first_public_at,
+      "dateModified" => presenter.public_updated_at,
+      "description" => presenter.description,
+      "publisher" => {
+       "@type" => "Organization",
+       "name" => "GOV.UK",
+       "logo" => {
+         "@type" => "ImageObject",
+         # TODO: change this to a better image, without the URL hard coded.
+         "url" => "https://assets.publishing.service.gov.uk/static/opengraph-image-a1f7d89ffd0782738b1aeb0da37842d8bd0addbd724b8e58c3edbc7287cc11de.png",
+       }
+      },
+      "image" => [
+        image["url"],
+      ],
+      "author" => {
+        "@type" => "Organization",
+        "name" => publishing_organisation["title"],
+      },
+    }
```

Comment:
I'm no expert on this linked data stuff, but I'd have expected to find at least some URIs in this... maybe that's a future improvement.